### ISSUE TITLE
Cryptic error message on geom_boxplot with orientation="y"

### ISSUE THREAD

**Issue by IKupriyanov-HORIS** [JetBrains/lets-plot#600]
Example:
```
data = {
    "alphabet": ["a","a","b","a","a","a","b","b","b","a","a","a"],
    "coeff": [
        0.9898989898989898, 0.98989898989899, 0.9871794871794872,
        0.9916666666666667, 0.9882352941176471, 0.9947368421052631,
        0.9916666666666667, 0.9882352941176471, 0.9947368421052631,
        0.9916666666666667, 0.9882352941176471, 0.9947368421052631
     ]
}
ggplot(data) + geom_boxplot(aes("alphabet", "coeff"), orientation="y")
```
Result:
```
Internal error: IllegalStateException : Not a collections of Double(s)
```

**Comment by alshan** [comment]
Related: #558 - automatic detection of orientation.

**Comment by RYangazov** [comment]
Fixed via e7a6e0d26c8d76458b68d1b13daa2eb3cd285373
